lib/fog/kubevirt/compute/compute.rb

```diff
@@ -472,6 +476,7 @@ def obtain_ssl_options(options)
 
             cert_store = OpenSSL::X509::Store.new
             certs.each do |cert|
+              raise ::Fog::Kubevirt::Errors::ValidationError, "Certificate has been expired" if cert_expired?(cert)
```

Comment:
while there is nothing wrong with validating the cert, IMO it should be handled by the the client library we're using (kubeclient) that uses net/http for communication with cluster. 
pls try to reproduce the issue by using kubeclient directly with the expired certificate and file an issue against that project.
we can merge this PR for now, but this just verifies one aspect of the certificate validity (and revert the PR once kubeclient handles it)

Reply:
I tried reproducing the issue using  kubeclient directly and it creates the client although  the certificate is expired. I will file an issue for it in the kubeclient project

Reply:
@masayag take a look at the issue I opened and what Beni Paskin-Cherniavsky answered. 
https://github.com/abonas/kubeclient/issues/514#issuecomment-911502476